server/data/personRecordApiClient.test.ts

```diff
@@ -1,18 +1,18 @@
 import nock from 'nock'
 import type { AuthenticationClient } from '@ministryofjustice/hmpps-auth-clients'
-import ExampleApiClient from './exampleApiClient'
+import PersonRecordApiClient from './personRecordApiClient'
 import config from '../config'
 
-describe('ExampleApiClient', () => {
-  let exampleApiClient: ExampleApiClient
+describe('PersonRecordApiClient', () => {
+  let personRecordApiClient: PersonRecordApiClient
   let mockAuthenticationClient: jest.Mocked<AuthenticationClient>
 
   beforeEach(() => {
     mockAuthenticationClient = {
       getToken: jest.fn().mockResolvedValue('test-system-token'),
     } as unknown as jest.Mocked<AuthenticationClient>
 
-    exampleApiClient = new ExampleApiClient(mockAuthenticationClient)
+    personRecordApiClient = new PersonRecordApiClient(mockAuthenticationClient)
```

Comment:
I see it was originally like this - strange not to simply define and initialise it in one line here.